WordPress/src/main/res/values/strings.xml

```diff
@@ -3864,22 +3864,8 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
 
     <!-- About -->
     <string name="about_blog">Blog</string>
-    <string name="about_automattic_app_name">About Automattic</string>
     <string name="about_automattic_main_page_title">About %1$s</string>
     <string name="about_automattic_legal_page_title">Legal and More</string>
     <string name="about_automattic_acknowledgements_page_title">Acknowledgements</string>
     <string name="about_automattic_version_label">Version %1$s</string>
-    <string name="about_automattic_share_with_friends">Share with friends</string>
-    <string name="about_automattic_rate_us">Rate us</string>
-    <string name="about_automattic_instagram">Instagram</string>
-    <string name="about_automattic_twitter">Twitter</string>
-    <string name="about_automattic_legal_and_more">Legal and more</string>
-    <string name="about_automattic_family">Automattic family</string>
-    <string name="about_automattic_work_with_us">Work with us</string>
-    <string name="about_automattic_work_from_anywhere">Work from anywhere</string>
-    <string name="about_automattic_terms_of_service">Terms of service</string>
-    <string name="about_automattic_privacy_policy">Privacy policy</string>
-    <string name="about_automattic_california_privacy_notice">California privacy notice</string>
-    <string name="about_automattic_source_code">Source code</string>
-    <string name="about_automattic_acknowledgements">Acknowledgements</string>
```

Comment:
@ravishanker Are we sure about those entries being deleted?

Did they stop being needed starting 0.0.3 or 0.0.4 of the lib? Feels suspicious to me 🤔

Reply:
Actually since these entries are copied from [the library's own `strings.xml` file](https://github.com/Automattic/about-automattic-android/blob/0.0.4/library/src/main/res/values/strings.xml) by our release scripts, those should actually not be modified manually in PRs. So this change to the strings needs to be reverted.

Reply:
After further investigation: those indeed need to be removed, but NOT because they're now "moved to the library" as the commit implied, but rather because the keys got renamed in the lib (as part of https://github.com/Automattic/about-automattic-android/commit/ea6045821722c67b4235fd7121d5ac619dac4a1c) during 0.0.4 and the old key names are unused (even in the library itself).